.chloggen/v1beta1-webhook.yaml

```diff
@@ -0,0 +1,21 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: collector
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Enable reconciliation of Collector v1beta1 CRD.
+
+# One or more tracking issues related to the change
+issues: [2620]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: | 
+  Users are expected to migrate to `otelcol.v1beta1.opentelemetry.io`. 
+  The support for `otelcol.v1alpha1.opentelemetry.io` will be removed in the future.
+  Follow https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version for version migration.
```

Comment:
i wonder if we should include our own documentation for how to do this. Specifically calling out the deprecated fields and the things to migrate. Especially when it comes to migrating the TA to the new TA crd as well.

Reply:
I think what we have might be good enough for now. However we should definitely talk about until when v1alpha1 will be served.